README.md

```diff
@@ -59,8 +59,31 @@ pull request and pushes commits onto three branches:
 
 ## Developer notes
 
+This project uses [Poetry](https://python-poetry.org/docs/#installation), so
+after you've installed Poetry itself, run this command in your clone of this
+repo to install all the dependencies you need for working on `ghstack`:
+```
+poetry install
```

Comment:
Should probably mention how to actually, ya know, run ghstack after you do poetry (unlike python setup.py develop it won't dunk a ghstack binary in your bin)

Reply:
Good point! Fixed.